verilog/formatting/formatter_test.cc

```diff
@@ -1168,6 +1168,62 @@ static const std::initializer_list<FormatterTestCase> kFormatterTestCases = {
      "  task t(int ii, bit bb);\n"
      "  endtask\n"
      "endclass\n"},
+    {"class c;virtual task t ( int  ii ,bit  bb );endtask endclass",
+     "class c;\n"
+     "  virtual task t(int ii, bit bb);\n"
+     "  endtask\n"
+     "endclass\n"},
+    {"class c;virtual task check (core_t core , "
+     "string error_msg , int  timeout );endtask;endclass;",
+     "class c;\n"
+     "  virtual task check(core_t core,\n"
+     "                     string error_msg,\n"
+     "                     int timeout);\n"
+     "  endtask;\n"
+     "endclass;\n"},
+    {"class c;virtual task check (core_t core , "
+     "string error_msg  );endtask;endclass;",
+     "class c;\n"
+     "  virtual task check(core_t core,\n"
+     "                     string error_msg);\n"
+     "  endtask;\n"
+     "endclass;\n"},
+    {"class c;virtual task check (core_t core  "
+     ");endtask;endclass;",
+     "class c;\n"
+     "  virtual task check(core_t core);\n"
+     "  endtask;\n"
+     "endclass;\n"},
+    {"class c;virtual task check (  "
+     ");endtask;endclass;",
+     "class c;\n"
+     "  virtual task check();\n"
+     "  endtask;\n"
+     "endclass;\n"},
+    {"class c; virtual task check_next_core_status(core_status_t core_status,"
+     " string error_msg, int timeout); endtask; endclass;",
+     "class c;\n"
+     "  virtual task check_next_core_status(\n"
+     "    core_status_t core_status,\n"
+     "    string error_msg,\n"
+     "    int timeout);\n"
```

Comment:
These should indent +4 spaces relative to previous level to emulate wrapping.

Reply:
You mean always (task and functions)? For example (indent=2, wrap=4):
<pre>
function f(aa_t aa, bb_t bb, cc_t cc);
</pre>
should be indented like:
<pre>
[function f(]
>>>>[aa_t aa,]
>>>>[bb_t bb,]
>>>>[cc_t cc);]
</pre>
instead of:
<pre>
[function f(]
>>[aa_t aa,]
>>[bb_t bb,]
>>[cc_t cc);]
</pre>

Reply:
Yes, for port-lists in particular.  (Generally, elements inside parentheses.)

Reply:
Fixed